src/OpenToolkit.Graphics/ES20/Helper.cs

```diff
@@ -74,24 +74,14 @@ public static void LoadBindings(IBindingsContext context)
             }
         }
 
-        public static void ClearColor(Color color)
-        {
-            GL.ClearColor(color.R / 255.0f, color.G / 255.0f, color.B / 255.0f, color.A / 255.0f);
-        }
-
-        public static void ClearColor(Color4 color)
-        {
-            GL.ClearColor(color.R, color.G, color.B, color.A);
-        }
-
-        public static void BlendColor(Color color)
+        public static void ClearColor(Color4<Rgba> color)
         {
-            GL.BlendColor(color.R / 255.0f, color.G / 255.0f, color.B / 255.0f, color.A / 255.0f);
+            GL.ClearColor(color.X, color.Y, color.Z, color.W);
```

Comment:
color.`W`? 🤔

Reply:
Dont reply to this one right away, only after my comment about hiding xyz... in Color3.cs i think it was